build.sbt

```diff
@@ -47,4 +47,6 @@ releaseProcess := Seq[ReleaseStep](
   setNextVersion,
   commitNextVersion,
   pushChanges
-)
\ No newline at end of file
+)
+
+Compile / doc / sources := Nil
```

Comment:
What's the purpose of this?

Reply:
See https://github.com/lampepfl/dotty/issues/15288#issuecomment-1324738643